maas-region/charmcraft.yaml

```diff
@@ -84,6 +84,24 @@ actions:
     description: List charmed MAAS controllers
   get-api-endpoint:
     description: Get MAAS API URL
+  create-backup:
+    description: Create a MAAS backup
```

Comment:
Maybe we should mention that this is noop for now in the description just in case anyone tries to use the actions before they are ready